MetadataExtractor/Formats/Png/PngMetadataReader.cs

```diff
@@ -102,14 +102,7 @@ public static
 
             foreach (var chunk in chunks)
             {
-                try
-                {
-                    directories.AddRange(ProcessChunk(chunk));
-                }
-                catch (Exception e)
-                {
-                    Debug.WriteLine(e);
-                }
+                directories.AddRange(ProcessChunk(chunk));
```

Comment:
There are various problems that can happen in ProcessChunk, so my vote is to let the exception happen and try to fix it afterwards.